### PR TITLE
Handle nullable outputs in job store simpletons.

### PR DESCRIPTION
**Pull request by mcovarr**
Some updates to support migrating null outputs to the job store simpleton table.  Also cherry picked a TES commit to get the build green.

### CODE REVIEW

File: database/migration/src/main/resources/changesets/restart_and_recover_migration.xml [hunk 1]
```diff
@@ -76,6 +76,23 @@
...
+        <preConditions onFail="MARK_RAN">
+            <tableExists tableName="JOB_STORE_RESULT_SIMPLETON"/>
+        </preConditions>
+        <comment>
+            This changeset is a retroactive insertion into the changeset order to enable migration of
+            null output values in a 0.19 Cromwell installation to the job store simpleton table introduced
+            in pluggable backend Cromwell.  Because this is a retroactive insertion, the "tableExists"
+            precondition keeps this changeset from running on a database that has already been migrated to
+            the pluggable backend schema (this migration is unnecessary in a database that has already been
+            migrated to the pluggable backend schema since a later changeset also drops this constraint).
```
Comment (Horneth) [inline database/migration/src/main/resources/changesets/restart_and_recover_migration.xml]: The later migration dropping the constraint doesn't fail if it has already been dropped by this one ?

**Comment by Horneth** [comment]
LGTM 👍

File: database/migration/src/main/resources/changesets/restart_and_recover_migration.xml (hunk 1 above)
Comment (mcovarr) [inline database/migration/src/main/resources/changesets/restart_and_recover_migration.xml]: Good question, I'll continue running the migration on my staging clone to see if that's an issue.

**Comment by ruchim** [comment]
:+1:

File: database/migration/src/main/resources/changesets/restart_and_recover_migration.xml (hunk 1 above)
Comment (mcovarr) [inline database/migration/src/main/resources/changesets/restart_and_recover_migration.xml]: The later migration actually seems to be cool with this.  🎉

Comment (geoffjentry) [inline database/migration/src/main/resources/changesets/restart_and_recover_migration.xml]: hooray ㊗️

**Pull request merged**